File: 8_kyu/Calculate_BMI.py
def bmi(weight, height):
    index = float(weight)/height/height
    for limit, name in (
    (18.5,"Underweight"),
    (25.0,"Normal"),
    (30.0,"Overweight"),
    (None,"Obese")):
        if not limit or index <= limit:
            return name

File: 8_kyu/Calculate_BMI.r
require(dplyr)

bmi <- function(weight, height) {
  index <- weight/height**2
  case_when(
    index <= 18.5 ~ "Underweight",
    index <= 25.0 ~ "Normal",
    index <= 30.0 ~ "Overweight",
    TRUE          ~ "Obese"
  )
}